scripts/projects.js

```diff
@@ -9,22 +9,23 @@ function Projects (project){
   this.date = project.projectDate;
 }
```

Comment:
consider replacing constructor with [key][pro] pair to future proof any additions you may make to your tempting.